x-pack/plugins/monitoring/public/views/apm/instance/index.js

```diff
@@ -39,7 +40,9 @@ uiRoutes.when('/apm/instances/:uuid', {
       });
 
       super({
-        title: `APM - Instance`,
+        title: i18n('xpack.monitoring.apm.instanceRouteTitle', {
```

Comment:
Let's be consistent with other Monitoring PRs:
```suggestion
        title: i18n('xpack.monitoring.apm.instance.routeTitle', {
```